meta.json

```diff
@@ -8,9 +8,21 @@
       {
           "api": "rdk:component:sensor",
           "model": "viam:lorawan:sx1302-gateway",
-          "markdown_link": "README.md#attributes",
+          "markdown_link": "README.md#configure-the-sx1302-gateway",
           "short_description": "A sensor component for a sx1302 LoRaWAN gateway HAT."
         },
```

Comment:
may want to remove this from the meta.json now so users don't add it.

Reply:
Yeah let's do it!